test/operator-pick.spec.ts

```diff
@@ -87,6 +88,15 @@ describe('pick operator unit tests', () => {
     expect(output![0].contact.email).to.eq(data.contact.email);
   });
 
+  it('it should find properties at different depths', () => {
+    const pick = new PickOperator({ name: 'pick', properties: 'id,color' })
```

Comment:
suggestion: an array of strings as the value for `properties` may be more interpretable for API users rather than a delimited string

Reply:
It's a good suggestion.  PickOperatorOptions supports a string or an array of strings.  There are two tests for this in the suite:
- it should pick array properties from an object
- it should pick csv properties from an object

Reply:
Got it - I see it now.